shared/js/views.jsx

```diff
@@ -599,22 +599,100 @@ loop.shared.views = (function(_, mozL10n) {
       isLoading: React.PropTypes.bool.isRequired,
       mediaType: React.PropTypes.string.isRequired,
       posterUrl: React.PropTypes.string,
+      remoteCursorPosition: React.PropTypes.object,
       // Expecting "local" or "remote".
       srcMediaElement: React.PropTypes.object
     },
 
+    getInitialState: function() {
+      return {
+        cursorOffset: {
```

Comment:
We happen to use the offset data for the cursor, but the values are actually referring to the video letterboxing, so the name should probably be describing "what it is" as opposed to the current one usage.